builtin/providers/google/resource_compute_region_instance_group_manager_test.go

```diff
@@ -0,0 +1,249 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"google.golang.org/api/compute/v1"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccRegionInstanceGroupManager_basic(t *testing.T) {
+	var manager compute.InstanceGroupManager
+
+	template := fmt.Sprintf("igm-test-%s", acctest.RandString(10))
+	target := fmt.Sprintf("igm-test-%s", acctest.RandString(10))
+	igm1 := fmt.Sprintf("igm-test-%s", acctest.RandString(10))
+	igm2 := fmt.Sprintf("igm-test-%s", acctest.RandString(10))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckRegionInstanceGroupManagerDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccRegionInstanceGroupManager_basic(template, target, igm1, igm2),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckRegionInstanceGroupManagerExists(
+						"google_compute_region_instance_group_manager.igm-region-basic", &manager),
+					testAccCheckRegionInstanceGroupManagerExists(
+						"google_compute_region_instance_group_manager.igm-region-no-tp", &manager),
+				),
+			},
+		},
+	})
+}
+
+func TestAccRegionInstanceGroupManager_update(t *testing.T) {
+	var manager compute.InstanceGroupManager
+
+	template1 := fmt.Sprintf("igm-test-%s", acctest.RandString(10))
+	target := fmt.Sprintf("igm-test-%s", acctest.RandString(10))
+	igm := fmt.Sprintf("igm-test-%s", acctest.RandString(10))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckRegionInstanceGroupManagerDestroy,
+		Steps: []resource.TestStep{
```

Comment:
I may be reading this wrong, but with only one step, I'm not sure this is actually updating anything--I think it's just creating a new resource with the desired config.